### DISCUSSION TITLE
Recommended settings for "euler_pp" sampler?

### DISCUSSION THREAD

**Discussion by not-ski** [Comfy-Org/ComfyUI#3861] [Q&A]
I see that the alternative CFG++ approach was added as `euler_pp` in commit https://github.com/comfyanonymous/ComfyUI/commit/69d710e40f43b87dd7124b9ee1e8982f987827e9.

How exactly does this differ from the standard CFG++ implementation, and what settings would you recommend (@comfyanonymous)? Thanks!

**Comment by brbbbq** [comment]
Here's a video outlining some of the new samplers, including Euler CFG++:
https://youtu.be/-GXJDz8i-Wo

And here's the project page for CFG++:
https://cfgpp-diffusion.github.io/
